### PR TITLE
Do not call actions with `state`

### PR DESCRIPTION
**Pull request by Andarist**

### CODE REVIEW

File: packages/core/test/assign.test.ts [hunk 1]
```diff
@@ -261,94 +261,62 @@ describe('assign', () => {
...
+var value = globalThis.fetch;
+Object.defineProperty(globalThis, 'fetch', {
+  set(newValue) {
+    debugger;
+    value = newValue;
+  },
+  get() {
+    return value;
+  }
+});
```
Comment (davidkpiano) [inline packages/core/test/assign.test.ts:273]: Hm?

**Review by davidkpiano — COMMENTED** [review]
LGTM, but I think there's some leftover debugging code

File: packages/core/test/assign.test.ts (hunk 1 above)
Comment (Andarist) [reply packages/core/test/assign.test.ts]: XDDDD haha, ye - let's pretend that this didn't happen :D

**Comment by Andarist** [comment]
@davidkpiano removed this leftover

**Review by davidkpiano — APPROVED** [review]

**Pull request merged**